.rubocop.yml

```diff
@@ -1,10 +1,11 @@
 AllCops:
-# Exclude/Include params should lie in the including .rubocop file
-# Ref. http://rubocop.readthedocs.io/en/latest/configuration/#inheritance
-#  Exclude:
-#    - 'db/schema.rb'
-#    - bin/**
-
+  # Exclude automatically generated rails files for easier app:update's etc
+  Exclude:
+    - bin/*
+    - db/schema.rb
+    - config/boot.rb
+    - config/environments/*
```

Comment:
I'm not 100 on this one as, while it is autogenerated, they also tend to be modified manually. 

The other ones are no-brainers though. :+1:

Reply:
On topic "modified manually" 😄 : I made it a habit to put the comment `# end of autogenerated file` on the bottom of the file and then do all manual modifications _beneath_ the comment. Sometimes you have to comment out a default setting, but apart from this I leave the autogenerated part above the comment completely untouched. This makes upgrading so much easier because you can see instantly in the diff which default settings change when doin the version upgrade. Just a quick tip.

Reply:
Yeah I think that's a good approach. While true that we may have some minor styling issues in those files by ignoring them I don't think that would be the end of the world. Changes still have to pass through code review and it's highly unlikely that we have complicated / hard to read code inside the environment files.